tests/cases/py_client/lib_test.py

```diff
@@ -92,6 +92,19 @@ def tearDown(self):
 
         base.TestCase.tearDown(self)
 
+    def testSession(self):
+        @httmock.urlmatch(path=r'.*/describe$')
```

Comment:
Please ignore, I didn't realize `re.match` treated the beginning of string and end of string differently for whatever reason.